src/main/index.ts

```diff
@@ -88,10 +88,18 @@ function initializeApp() {
       mainWindow.destroy();
     });
 
+    ipcMain.on(channels.window.reload, () => {
+      mainWindow.reload();
+    });
+
     ipcMain.on(channels.window.isMaximized, (event) => {
       event.sender.send(channels.window.isMaximized, mainWindow.isMaximized());
     });
 
+    ipcMain.on(channels.openPath, (_, value: string) => {
+      shell.openPath(value).then();
```

Comment:
What's the purpose of creating a promise here that's never used?

Reply:
I usually do `void shell.openPath(value);` to dismiss the promise itself.

Reply:
As this part of the code is 1,5 years old I don't exactly remember anymore what the reason was😄 but I believe it was to get around an unhandled promise error. (Which doesn't seem to be the case anymore, as I tried all variations). Also, there are more empty then() methods in the code.  I can change it to holland's suggestion.